client/dashboard/customizable.js

```diff
@@ -252,17 +264,19 @@ class CustomizableDashboard extends Component {
 			primaryDate,
 			secondaryDate,
 		};
+		const showTaskList =
```

Comment:
We have similar logic above to check if the task list is enabled.  Maybe we can move these into a reusable method of `isTaskListEnabled()`.

Also, I think we can remove the check for `wcSettings.onboarding` here.

Reply:
I've removed the `wcSettings.onBoarding` check but I think the logic differs enough that extracting it out into another function is premature, otherwise there's going to be some very funky logic happening in the extracted function.